futile.lua

```diff
@@ -0,0 +1,32 @@
+isPhoneOpen = false
+
+CreateThread(function()
+    while true do
+        Wait(5)
```

Comment:
This has to be every frame. Especially on slow PC’s.
```suggestion
        Wait(0)
```